main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala

```diff
@@ -51,10 +51,11 @@ object ParsedAst {
       *
       * @param sp1   the position of the first character in the declaration.
       * @param name  the name of the namespace.
+      * @param uses  the uses in the abstract syntax tree.
```

Comment:
Are these the uses that occur anywhere inside the namespace?

If so, maybe we should just call it "the uses available in the namespace".